plugins/microsoft-calendar/src/components/CalendarEventPopoverContent.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { sortBy } from 'lodash';
+import React from 'react';
+import DOMPurify from 'dompurify';
+
+import { Link } from '@backstage/core-components';
+import {
+  Box,
+  Divider,
+  IconButton,
+  Tooltip,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
+
+import { AttendeeChip } from './AttendeeChip';
+import { MicrosoftCalendarEvent } from '../api';
+import { getTimePeriod, getOnlineMeetingLink } from './util';
+
+const useStyles = makeStyles(
+  theme => ({
+    description: {
+      wordBreak: 'break-word',
+      '& a': {
+        color: theme.palette.primary.main,
+        fontWeight: 500,
+      },
+    },
+    divider: {
+      marginTop: theme.spacing(2),
+      marginBottom: theme.spacing(2),
+    },
+  }),
+  {
+    name: 'GCalendarEventPopoverContent',
```

Comment:
All these names need updating properly. `GCalendar` is copy paste from the `GoogleCalendar`. Should be `MicrosoftCalendar` etc..

Reply:
made the updates. thank you for all the points that you both mentioned @benjdlambert  @awanlin , i really appreciate your efforts